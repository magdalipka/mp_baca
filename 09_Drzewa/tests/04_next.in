1
8
ENQUE 35 Andrzej Wolny
ENQUE 30 Andrzej Inny
ENQUE 40 Iwona Wolny
ENQUE 45 Iwona Inny
NEXT 30
NEXT 35
NEXT 40
NEXT 45
