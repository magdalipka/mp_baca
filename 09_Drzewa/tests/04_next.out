ZESTAW 1
NEXT 30: 35 - Andrzej Wolny
NEXT 35: 40 - Iwona Wolny
NEXT 40: 45 - Iwona Inny
NEXT 45: BRAK
